### ISSUE TITLE
Detailing Daily Weather Conditions using Graphs Suns

### ISSUE THREAD

**Issue by Sudhanshutiwari-cs** [Mohit5Upadhyay/WeatherApplication#42]
Current Weather Report can be changed to Hours report using graphs and suns like animations..Hence Therefore Assign me this

**Comment by sambedan1** [comment]
I will create visually appealing graphs to represent daily weather conditions, such as temperature, humidity, and wind speed. Additionally, I will incorporate sun icons to indicate sunny days, which will help users quickly grasp the weather forecast at a glance.

Please assign this task to me so I can implement these enhancements

**Comment by Mohit5Upadhyay** [comment]
Yes , @sambedan1 and @Sudhanshutiwari-cs  go ahead to implement graph functionality but they are dynamic in nature as per the current weather condition

**Comment by sambedan1** [comment]
ok thank you for assign me
